src/Rings/mpoly-affine-algebras.jl

```diff
@@ -33,8 +33,8 @@ If, say, `A = R/I`, where `R` is a multivariate polynomial ring over a field
 as a `K`-vector space, `false` otherwise.
 
 !!! note 
-    `A` is finite-dimensional as a `K`-vector space iff it has Krull dimension zero. This condition is checked by the function.
-    
+    `A` is finite-dimensional as a `K`-vector space iff it has Krull dimension
+    less or equal zero. This condition is checked by the function.
```

Comment:
See also https://github.com/oscar-system/Oscar.jl/issues/5056 for a related discussion (but this is an obvious fix and so no need to wait for that discussion to be resolved)